### PR TITLE
Update Laravel quickstarts to reference latest SDK changes

### PR DESCRIPTION
**Pull request by evansims**
This PR includes changes to the Laravel quickstart to fix errors and bring the documentation up to date with the latest version of Laravel and our Laravel SDK.

### CODE REVIEW

File: articles/quickstart/webapp/laravel/01-login.md [hunk 1]
```diff
@@ -56,41 +56,59 @@ First, we need to add the Auth0 Services to the list of Providers in `config/app
...
+      'aliases' => [
+          // ...
+          'Auth0' => Auth0\Login\Facade\Auth0::class,
+      ],
 ```
 
 Finally, you will need to bind a class that provides the app's User model each time a user is logged in or a JWT is decoded. You can use the `Auth0UserRepository` provided by this package or build your own (see the "Custom User Handling" section below).
 
-Add the following to your `AppServiceProvider::register()` method:
+Replace the contents of that file with the following:
```
Comment (frederikprijck) [inline articles/quickstart/webapp/laravel/01-login.md]: What does `that file` mean here?

File: articles/quickstart/webapp/laravel/01-login.md [hunk 2]
```diff
@@ -56,41 +56,59 @@ First, we need to add the Auth0 Services to the list of Providers in `config/app
...
-];
+<?php
+
+    return [
+        // ...
+
+      'aliases' => [
+          // ...
+          'Auth0' => Auth0\Login\Facade\Auth0::class,
+      ],
```
Comment (frederikprijck) [inline articles/quickstart/webapp/laravel/01-login.md]: Is it expected that we are:

- opening the php tag but not closing it ?
- opening an array but not closing it ?

Comment (evansims) [reply articles/quickstart/webapp/laravel/01-login.md]: PHP tags are never closed in files that are entirely PHP, that's alright.

This section of the quickstart is suggesting readers 'add an alias in the same file' and not replace the contents of that file with content, so we don't need to rewrite all of the code in this portion of the QS or make it fully syntactically accurate, I think. We're just indicating the relevant milestones in the code so readers can find the position of the position in the file so readers can find it.

File: articles/quickstart/webapp/laravel/01-login.md (hunk 1 above)
Comment (evansims) [reply articles/quickstart/webapp/laravel/01-login.md]: The file path is indicated in the source code markup that follows that line, app/Providers/AppServiceProvider.php, but I'll reword that line to make it clearer.

File: articles/quickstart/webapp/laravel/01-login.md (hunk 2 above)
Comment (frederikprijck) [reply articles/quickstart/webapp/laravel/01-login.md]: > PHP tags are never closed in files that are entirely PHP, that's alright.

TIL! Thanks for explaining!

**Review by frederikprijck — APPROVED** [review]

**Pull request merged**